tests/test_awsdatabricks.py

```diff
@@ -0,0 +1,597 @@
+import copy
+import io
+from datetime import datetime
+from unittest.mock import patch
+from uuid import uuid4
+
+import boto3 as boto
+from botocore.response import StreamingBody
+from botocore.stub import Stubber
+
+from sync.awsdatabricks import create_prediction_for_run
+from sync.config import DatabricksConf
+from sync.models import Response
+
+MOCK_RUN = {
```

Comment:
I've always preferred defining mock data inline, if the size is not completely unwieldy